cluster-api-gcp/cloud/google/machineactuator.go

```diff
@@ -467,13 +481,21 @@ func (gce *GCEClient) updateAnnotations(machine *clusterv1.Machine) error {
 
 // The two machines differ in a way that requires an update
 func (gce *GCEClient) requiresUpdate(a *clusterv1.Machine, b *clusterv1.Machine) bool {
+	aConfig, aerr := gce.providerconfig(a.Spec.ProviderConfig)
```

Comment:
If you are just going to do a deep equal, why bother decoding? You can just compare the two strings.

Reply:
Good point.